blockade/net.py

```diff
@@ -355,7 +321,5 @@ def network_state(device):
         if " duplicate " in output:
             return NetworkState.DUPLICATE
         return NetworkState.NORMAL
-
-    except subprocess.CalledProcessError:
-        # TODO log error somewhere?
+    except:
```

Comment:
I think this shouldn't be so broad - `except Exception` maybe. iirc bare except will even catch stuff like `KeyboardInterrupt`